Kentico.Kontent.Delivery.Abstractions/SharedModels/IError.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Kentico.Kontent.Delivery.Abstractions
+{
+    /// <summary>
+    /// Represents a faulty JSON response from Kentico Kontent Delivery API.
+    /// </summary>
+    public interface IError
+    {
+        /// <summary>
+        /// Gets error Message.
+        /// </summary>
+        string Message { get; }
+
+        /// <summary>
+        /// Gets request id.
+        /// </summary>
+        string RequestId { get; }
+
+        /// <summary>
+        /// Gets error code.
```

Comment:
I'd be a bit more specific..what kind of error code? HTTP status? KK-specific error code?

Reply:
Could you please have a look now?